Sources/Workspace/Workspace.swift

```diff
@@ -340,13 +349,15 @@ public class Workspace {
     public func edit(
         dependency: ManagedDependency,
         packageName: String,
+        engine: DiagnosticsEngine,
```

Comment:
lets use `diagnostics` instead of engine as it isn't very good name. We need to fix this at other places too (but thats a separate thing)